test/functional/p2p_getaddr_caching.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test addr response caching
+"""
```

Comment:
nit: docstrings should be one line if there's only one line of text. Join these lines.